benchopt/plotting/html/static/main.css

```diff
@@ -272,3 +279,21 @@ td.number {
   height: 1000px;
 }
 
+.btn svg {
+  width: 1.5rem;
+  height: 1.5rem;
+  color: white;
+}
+
+.btn > * + * {
+  margin-left: 0.5rem;
+}
```

Comment:
Can you add a comment here too (not sure what `> * + *` mean actually :sweat_smile: )